beacon-chain/operations/attestations/metrics.go

```diff
@@ -30,6 +30,16 @@ var (
 		Name: "expired_block_atts_total",
 		Help: "The number of expired and deleted block attestations in the pool.",
 	})
+	attCount = promauto.NewGauge(
+		prometheus.GaugeOpts{
+			Name: "attestations_in_pool_total",
+			Help: "The number of attestations in the pool.",
+		},
+	)
+	expiredAtts = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "expired_atts_total",
+		Help: "The number of expired and deleted attestations in the pool.",
+	})
```

Comment:
There is no longer a need to have separate metrics for unaggregated and aggregated atts, so I created new metrics for the new design